web/src/test/kotlin/no/nav/su/se/bakover/web/routes/person/PersonRoutesKtTest.kt

```diff
@@ -50,6 +52,8 @@ internal class PersonRoutesKtTest {
     @Test
     fun `kan hente data gjennom PersonOppslag`() {
         val testIdent = "12345678910"
+        val fødselsdato = "1990-01-01"
+        val alder = fødselsdato.let { Period.between(LocalDate.parse(it), LocalDate.now()).years }
```

Comment:
Jeg synes generelt det er bedre med statiske data i tester, enn testkode som hermer etter implementasjonens logikk :man_shrugging: Hvis vi ikke bryr oss, kan vi like gjerne bruke fødselsdato.getAlder()

Reply:
Forslag til hvordan vi løser dette, uten å knekke testene ved årsskifter sånn som i EktefelleTest? Har vi noen overstyrt klokke e.l. som det går an å bruke i tester?

Reply:
Vi har overstyrte klokker i en del tester. Men for dataklasser som bruker now() er det litt mer styrete. Da har jeg endt med enten å sende inn *Date som en parameter eller bruke en Factory som setter en default verdi.

Dette er mer problematisk der det tryner på millisekunder, ikke på år :grin:

På en annen side prøver vi å gå vekk fra å bruke implementasjonene i routes-testene. Dvs. vi mocker servicen. Da kan man ha en fixedClock i testen og bruke samme verdi i mock og verify.